addons/account/models/account_journal_dashboard.py

```diff
@@ -990,9 +990,9 @@ def _select_action_to_open(self):
         elif self.type == 'cash':
             return 'action_view_bank_statement_tree'
         elif self.type == 'sale':
-            return 'action_move_out_invoice_type'
+            return 'action_move_out_invoice'
```

Comment:
You can't, it depends on a xml update, which is not done by default
You have to not change the name of these actions